### ISSUE TITLE
Buff Shades of Morton to be 3x faster than it is. Some data inside!

### ISSUE THREAD

**Issue by mylife212** [oldschoolgg/oldschoolbot#5125]
## 🚀 Feature Proposal
 
Iirc... When we were testing shades, the slow speed was decided to be kept in (Around 19~ shades burnt/hr) since there was little to no info available online. tldr at bottom

Been doing some Shades of mort'on in game, so finally have some rough data on the real speed. Apologies for the long post. 
Note:1 Sacred oil =Pyre log to burn shades. It takes 5 minutes to get 100% Sanctity at the temple on a mass world, which lets you create 25 Sacred oil. This is **300 Sacred oil/hr INCLUDING Banking** .If you have a Flamtaer bracelet, it is DOUBLED - this is not a boost for Shades in bot but worth noting. You can burn shades really fast, which came out to **22 Shades cremated every 3.5 minutes, including bank**. Since the Shades trip in bot assumes you make oil and burn the remains at same time, it is very slow compared to in-game.

I think that tripling the speed is more than fair enough, as the time to open chests does take quite a bit of time and thats instant due to /open. So making it 60 Shades burnt/hr is a great balance.

tldr; Triple the speed of Shade of Morton trips, so its around 60 Shades cremated/hr.
